day-4/ques-13.js

```diff
@@ -0,0 +1,23 @@
+//1Problem Statement: Write a program to return inverse of an array
+
```

Comment:
what is expected result? please comment out

Reply:
resolved